pkg/corerp/api/v20231001preview/util.go

```diff
@@ -132,6 +132,11 @@ func toOutputResourcesDataModel(outputResources []rpv1.OutputResource) []*Output
 			r.RadiusManaged = or.RadiusManaged
 		}
 
+		if *or.Status == rpv1.DeploymentStatusFailed {
+			r.Status = to.Ptr(DeploymentStatusFailed)
+		} else if *or.Status == rpv1.DeploymentStatusSucceeded {
+			r.Status = to.Ptr(DeploymentStatusSucceeded)
+		}
```

Comment:
What about the other statuses? I believe that we should convert all the statuses.

Reply:
What other statuses are there?